### PR TITLE
Migration guide page

### PR DESCRIPTION
**Pull request by scarletfog**
## Description
The migration guide -> from 7 to 8.

🚨 There something wrong with styling in the code blocks, I could not figure it out. Maybe @budnix knows?
🚨 I suppose it will need some adaptations/corrections, feel free to do it directly @wojciechczerniak. I did what I could before 🌴

### CODE REVIEW

**Comment by wszymanski** [comment]
> 🚨 There something wrong with styling in the code blocks, I could not figure it out. Maybe @budnix knows?

Changed.

File: tutorials/migration-guide.html [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+  // Trimming first row.
+  if (row < 9) {
+    return row + 1;
+  }
+
+  return null;
+}</code></pre>
+
+      <p>Now, if you want to get the same results you need to use the <code>TrimmingMap</code>:</p>
+      <pre><code class="lang-js">import { TrimmingMap } from "handsontable/es/translations";
```
Comment (budnix) [inline tutorials/migration-guide.html:120]: Hmm, maybe it's better to re-export that Maps in the `index.js` file as we already know today that `translations` module name is temporary. @wojciechczerniak

File: tutorials/migration-guide.html [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+      <p>
+          You can find the plugin's isMovePossible API method useful when you want to determine if the move action is
+          possible. <code>movePossible</code> parameter of <code>beforeRowMove</code>, <code>afterRowMove</code>,
+          <code>beforeColumnMove</code>, and <code>afterColumnMove</code> hooks may be helpful as well.
+      </p>
+
+      <h3>Changes in ManualColumnFreeze plugin behavior</h3>
+
+      <p>
+          The <code>ManualColumnFreeze</code> plugin itself also works differently. Before the <strong>8.0.0v</strong>
```
Comment (budnix) [inline tutorials/migration-guide.html:311]:
```suggestion
          The <code>ManualColumnFreeze</code> plugin itself also works differently. Before the <strong>v8.0.0</strong>
```
I think that the `v` should be placed as a prefix. It looks more natural.

File: tutorials/migration-guide.html (hunk 1 above)
Comment (wojciechczerniak) [reply tutorials/migration-guide.html:120]: Will we export them in the end? Your idea was to use a registry and initialize maps with string names, so they can be used from the UMD build as well.

Comment (budnix) [reply tutorials/migration-guide.html:120]: True, I assume that only BaseMap will be necessary for export (needed for custom Maps?).

File: tutorials/migration-guide.html (hunk 2 above)
Comment (budnix) [reply tutorials/migration-guide.html:311]:
```suggestion
          The <code>ManualColumnFreeze</code> plugin itself also works differently. Before the 8.0.0
```
Or this. As the migration guide uses mostly simplified notation (without prefix and using regular font style) so it's more consistent with the overall document. Unless the emphasizing of the version, in this case, was intentional.

**Comment by budnix** [comment]
Another suggestion that comes to my mind is a proposition for creating a new category `Migration` where under the category will be a file `migration-from-7-to-8`. The solution seems to be open for future migrations.

It doesn't have to be done now. But if we do this before release, we will be able to avoid unnecessary redirects in the future.

**Comment by wojciechczerniak** [comment]
> Another suggestion that comes to my mind is a proposition for creating a new category `Migration` where under the category will be a file `migration-from-7-to-8`. The solution seems to be open to future new migrations.
> 
> It doesn't have to be done now. But if we do this before release, we will be able to avoid unnecessary redirects in the future.

Our oryginal proposition was like that but Krzysztof has some plans for the docs and prefered this way.

File: tutorials/migration-guide.html (hunk 1 above)
Comment (wojciechczerniak) [reply tutorials/migration-guide.html:120]: Would be helpful, but is not necessary. Recreating all the basic methods could be tedious though.

**Pull request merged**